x11-plugins/gkrellm-cpupower/files/gkrellm-cpupower-0.2-makefile.patch

```diff
@@ -0,0 +1,45 @@
+--- a/Makefile
++++ b/Makefile
+@@ -1,22 +1,29 @@
+ # Makefile for gkrellm cpupower plugin
+ 
+-GTK_INCLUDE = `pkg-config gtk+-2.0 --cflags`
+-GTK_LIB = `pkg-config gtk+-2.0 --libs`
++PKG_CONFIG = pkg-config
+ 
+-FLAGS = -O2 -Wall -fPIC $(GTK_INCLUDE)
++GTK_INCLUDE = $(shell ${PKG_CONFIG} gtk+-2.0 --cflags)
++GTK_LIB = $(shell ${PKG_CONFIG} gtk+-2.0 --libs)
```

Comment:
@thesamesam This is what I figured was the "correct" way.

Reply:
PKG_CONFIG ?= ..., no?

Reply:
Yes :-) Changed.